test/hotspot/jtreg/compiler/ccp/TestShiftConvertAndNotification.java

```diff
@@ -38,7 +38,7 @@
  * @bug 8313672
  * @summary Test CCP notification for value update of AndL through LShiftI and
  *          ConvI2L (no flags).
- * @run driver compiler.ccp.TestShiftConvertAndNotification
+ * @run main/othervm compiler.ccp.TestShiftConvertAndNotification
```

Comment:
Can we just use `main` instead of `main/othervm`?

Reply:
Let's try (adjusted).